.github/workflows/release.yml

```
@@ -150,21 +172,59 @@ jobs:
           VERSION="${GITHUB_REF_NAME}"
-          VER_CLEAN="${VERSION#v}"   # strip leading 'v': v1.4.0 → 1.4.0
+          VER_CLEAN="${VERSION#v}"   # strip leading 'v': v1.4.1 → 1.4.1
+
           # Substitute the __APP_VERSION__ placeholder in the template
           sed "s/__APP_VERSION__/${VER_CLEAN}/g" \
             .github/windows-installer.nsi > installer.nsi
+
+          # MUI2 README page — must exist at makensis compile time.
+          # Plain text is sufficient; MUI2 renders it in a scrollable text box.
+          cat > readme.txt << 'EOF'
+DeepDiff DB - Quick Start
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)